.env

```diff
@@ -5,7 +5,7 @@ STACK_NAME=app-giskard-ai
 # Backend
 BACKEND_CORS_ORIGINS=["http://localhost","http://localhost:4200","http://localhost:3000","http://localhost:8080","http://localhost:19000"]
 PROJECT_NAME=Giskard
-SECRET_KEY=
+SECRET_KEY=5af086f91f8c0a195701a7c418633353b7eb65ef38d7c7de30abd63ef3dd2da3d1514bf0ce14423fc79a2d35541323ea90ba6e6fc988378a54f1cc2ae08702b1
```

Comment:
A secret in clear?

Reply:
Hi @CyrilLeMat ! Thanks for pointing it out. 
It's still under construction and is hardcoded only for the sake of simplicity. There's a card in our backlog not to forget to remove it.